src/include/baselib/messaging/GraphQLErrorHelpers.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * This file is part of the swblocks-baselib library.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __BL_MESSAGING_GRAPHQLERRORHELPERS_H_
+#define __BL_MESSAGING_GRAPHQLERRORHELPERS_H_
+
+#include <baselib/data/eh/ServerErrorHelpers.h>
+#include <baselib/data/DataModelObject.h>
+#include <baselib/data/DataModelObjectDefs.h>
+#include <baselib/data/models/ErrorHandling.h>
+
+#include <baselib/core/CPP.h>
+#include <baselib/core/ObjModel.h>
+#include <baselib/core/BaseIncludes.h>
+
+#include <baselib/messaging/BrokerErrorCodes.h>
+
+namespace bl
+{
+    namespace messaging
+    {
+        template
+        <
+            typename E = void
+        >
+        class GraphQLErrorHelpersT
+        {
+            bl::eh::void_exception_callback_t m_defaultCallback = bl::dm::ServerErrorHelpers::defaultEhCallback();
```

Comment:
Why is this private member necessary? You can just use bl::dm::ServerErrorHelpers::defaultEhCallback() for the default value below.

Reply:
btw, bl:: prefix also not necessary